pkg/provision/providers/qemu/launch.go

```diff
@@ -323,7 +323,6 @@ func launchVM(config *LaunchConfig) error {
 		"-no-reboot",
 		"-boot", fmt.Sprintf("order=%s,reboot-timeout=5000", bootOrder),
 		"-smbios", fmt.Sprintf("type=1,uuid=%s", config.NodeUUID),
-		"-smbios", fmt.Sprintf("type=11,value=io.systemd.stub.kernel-cmdline-extra=%s", config.sdStubExtraCmdline),
```

Comment:
this was broken already, since we append the `sdStubExtraCmdlineConfig` later down below only, so first boot always missed `talos.config`